create-plugin-update/action.yml

```diff
@@ -25,7 +25,8 @@ runs:
 
```

Comment:
are any changes in this file necessary? (changes look like for debugging purposes)